slices/ca-certificates.yaml

```diff
@@ -16,6 +16,25 @@ slices:
       ]
       content.write("/etc/ssl/certs/ca-certificates.crt", "".join(certs))
 
+  default-certs:
+    essential:
+      - ca-certificates_data
+    contents:
+      /etc/ca-certificates.conf: {text: FIXME, mutable: true}
+      /usr/share/ca-certificates/mozilla/**:
+    mutate: |
+      certs_dir = "/usr/share/ca-certificates/mozilla/"
+      names = [
+        "mozilla/{}".format(name) for name in content.list(certs_dir)
+      ]
+      content.write("/etc/ca-certificates.conf", "\n".join(names))
+
+  bins:
```

Comment:
do we need or use this somewhere? was it requested? 

If not, please remove it.

Reply:
So, yes, there have been some requests for having the `update-ca-certificates` binary available in a slice: https://github.com/dotnet/dotnet-docker/issues/5045.

In any case, since this is not part of the `data` slice, I think it's okay to provide the `update-ca-certificates` so that people can use it as they wish.

Reply:
Good catch. As this file will anyways be generated by `update-ca-certificates`, there is indeed no need to keep it here.

Reply:
I think he was talking about the `bins` slice. But yeah, I do agree with the `ca-certificates.conf` resolution.